### ISSUE TITLE
password.txt?

### ISSUE THREAD

**Issue by zhang1frank** [AXLEproject/axle-healthcare-benchmark#2]
Several parts of the code check for a password.txt file that is supposed to be in cda-generator. Such a file does not come with cloning the git repository, and I see that is in fact listed in the gitignore file. Is there a way to generate, or copy-paste the file onto one's local machine? It doesn't seem one can properly install the generator without it.

**Comment by yhavinga** [comment]
This is correct - please email info@portavita.nl with a request for the AXLE synthetic dataset password.

**Comment by mukundesh** [comment]
I have emailed the above email address for password, but haven't heard back. Is there any other way for getting the password ?
